integrationtests/tests/tests.rs

```diff
@@ -722,6 +726,34 @@ async fn set_lightning_invoice_expiry() -> Result<()> {
     task_group.shutdown_join_all().await
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn client_cannot_pay_expired_invoice() -> Result<()> {
```

Comment:
This test is a sketch of checking if the expiry is enforced, but it doesn't work yet. The invoice expiry it set to 1 second, and it sleeps for 2 seconds so the invoice should be expired when fund_outgoing_ln_contract is called

Reply:
Fixed test in https://github.com/fedimint/fedimint/pull/1074/commits/d1bf57e966283cd30cb3222fb307735dd9585c3e